Kernel/Arch/x86/common/Boot/boot.S

```diff
@@ -177,9 +177,9 @@ start:
     mov $PROTECTED_MODE_16_BIT_CODE, %edi     /* destination address of the code */
     rep movsb
 
-    /* Copy real_mode to continue to low memory section  */
+    /* Copy real_mode to end_of_print_and_halt_function to low memory section  */
     lea real_mode, %eax
-    lea continue, %ebx
+    lea end_of_print_and_halt_function, %ebx
     sub $0xc0000000, %ebx
```

Comment:
```suggestion
    mov $(end_of_print_and_halt_function - KERNEL_BASE), %ebx
```

`<AK/Sections.h>` provides the `KERNEL_BASE` macro which means we don't need to hard-code `0xc0000000` everywhere. :)